stubs/jsonschema/jsonschema/__main__.pyi

```diff
@@ -0,0 +1 @@
+from jsonschema.cli import main as main
```

Comment:
I believe that the addition of this file is what is causing stubtest to fail. Due to the special behavior of `__main__`, we should just leave this file out of the stubs.